client/src/components/PhotoCarousel.jsx

```diff
@@ -38,46 +68,32 @@ export default class PhotoCarousel extends Component {
   };
 
   render() {
-    const { images, showModal } = this.state;
+    const { mediaData } = this.state;
+    const media = mediaData.slice(0, 10).map((image, index) => {
+      if (index % 3 === 0) {
+        return (
+          <div className="media" style={{ minWidth: '0', margin: '2px' }}>
+            <img src={image} alt="" style={imgStyle} />
+          </div>
+        );
+      }
+      if (index % 3 === 1) {
+        return (
+          <div
+            id="smallMedia"
+            className="media-column-container"
+            style={{ minWidth: '0', margin: '2px' }}
+          >
+            <img src={image} alt="" style={imgColStyle} />
```

Comment:
Really awesome work on this! Consider adding something like this line to add the second column style image: `<img src={mediaData[index + 1]} alt="" style={imgColStyle} />`

Reply:
Added!